Doc/library/os.path.rst

```diff
@@ -341,6 +341,13 @@ the :mod:`glob` module.)
    that contains symbolic links.  On Windows, it converts forward slashes to
    backward slashes. To normalize case, use :func:`normcase`.
 
+  .. note::
+      On POSIX systems, in accordance with `IEEE Std 1003.1 2013 Edition; 4.12
+      Pathname Resolution <http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_12>`_,
+      double-slashes at the start of a path are allowed and represent the root
+      directory, and more than two slashes at the start of a path are treated as
+      a single slash and normalized as such.
+
```

Comment:
It's section 4.13, not 4.12, and it should just quote the relevant part instead of interpret it (wrongly): "[i]f a pathname begins with two successive <slash> characters, the first component following the leading <slash> characters may be interpreted in an implementation-defined manner, although more than two leading <slash> characters shall be treated as a single <slash> character". 

For example, in MSYS2, Python uses posixpath and double slashes at the start are reserved for UNC paths:
```python
>>> sys.platform
'msys'
>>> posixpath is os.path
True
>>> os.path.normpath('///c/temp/spam')
'/c/temp/spam'
>>> os.path.normpath('//c/temp/spam')
'//c/temp/spam'
>>> os.path.exists('/c/temp/spam')
True
>>> os.path.exists('///c/temp/spam')
True
>>> os.path.exists('//c/temp/spam')
False
>>> os.path.exists('//localhost/c$/temp/spam')
True
```

Reply:
Thank you for your review! I made changes to PR